src/runtime/storage/volatile-storage.ts

```diff
@@ -445,7 +445,7 @@ export class VolatileSingleton extends VolatileStorageProvider implements Single
     return this._stored;
   }
 
-  async set(value : {id: string}, originatorId: string = null, barrier: string = null): Promise<void> {
+  async set(value, originatorId: string = null, barrier: string = null): Promise<void> {
```

Comment:
I wanted to use these objects directly in the new tests so I could access the clearItemsForTesting() method, and that made the compiler fail because this isn't even close to the right type.

`value` has at least an id but also various other dynamic fields. Note that the same API elsewhere is just left as implicit any (ie. Collection.store), so I just lined this up with those.

Ultimately I wanted minimal updates to this code because it's very soon to disappear anyway.

Reply:
I think I prefer this new version anyway :-)